build-scripts/bundle.cjs

```diff
@@ -1,7 +1,37 @@
 const path = require("path");
+const browserslist = require("browserslist");
+const lightningcss = require("lightningcss");
 const env = require("./env.cjs");
 const paths = require("./paths.cjs");
 
+const decoder = new TextDecoder();
+const cssTargets = lightningcss.browserslistToTargets(
+  browserslist.loadConfig({ path: ".", env: "modern" })
+);
```

Comment:
This should be based on if we are building for modern or es5.

Reply:
Hmm.
Well the thing is that Lightning CSS would start doing transpilation. While transpilation is good (say, we could throw out any other CSS transpilers we use, and start using modern CSS features here and now), if I understand correctly, there should be no need for transpilation (unless we already have some other transpiler).

FYI, here are some results from running Lightning CSS on `@material/mwc-textfield`:
`Original: 47.7kb`
|  | Default targets | Modern targets | Legacy targets |
|--------|--------|--------|--------|
| Minify off | 48.7kb | 49.9kb | 69kb |
| Minify on | 43.6kb | 44.5kb | 60.7kb |